todo_app/data/trello_items.py

```diff
@@ -0,0 +1,118 @@
+from functools import cached_property
+from urllib3.exceptions import InsecureRequestWarning
+import typing
+import requests
+import os
+
+
+# Suppress the warnings from urllib3
+requests.packages.urllib3.disable_warnings(category=InsecureRequestWarning)
+
+
+class Item:
+    ''' Class to represent an item (card)'''
+    def __init__(self, id, name, status='To Do'):
+        self.id = id
+        self.name = name
+        self.status = status
+
+    @classmethod
+    def from_trello_card(cls, card, list):
+        return cls(card['id'], card['name'], list['name'])
+
+
+class TrelloList:
+    ''' Class to represent a List'''
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
+
+    @classmethod
+    def from_trello_list(cls, list):
+        return cls(list['id'], list['name'])
+
+
+class TrelloAPI():
+    ''' Service Trello API related actions '''
+    def __init__(self) -> None:
+        self.API_KEY = os.environ.get('API_KEY')
+        self.API_TOKEN = os.environ.get('API_TOKEN')
+        self.BOARD_ID = os.environ.get('BOARD_ID')
+        self.cards : typing.List[Item] = []
+        self.lists : typing.List[TrelloList] = []
+
+        # call get_items to initialise in RAM cards and lists properties
+        self.get_items()
+
+    def _send_requst(self, url, method='GET', **query):
```

Comment:
Just a typo, but in a dynamic language I can guarantee somebody will trip up with this at some point!
```suggestion
    def _send_request(self, url, method='GET', **query):
```